packages/relay/tests/lib/eth/eth_sendRawTransaction.spec.ts

```diff
@@ -105,14 +112,15 @@ describe('@ethSendRawTransaction eth_sendRawTransaction spec', async function ()
         ethImpl.sendRawTransaction,
         false,
         ethImpl,
-        [txHash],
+        [txHash, getRequestId()],
       );
     });
 
     it('should return a computed hash if unable to retrieve EthereumHash from record due to contract revert', async function () {
       const signed = await signTransaction(transaction);
 
       restMock.onGet(`transactions/${transactionId}`).reply(200, null);
+      restMock.onGet(`network/exchangerate`).reply(200, mockedExchangeRate);
```

Comment:
Instead of adding it manually in every test, move this to a `beforeEach` block:
```typescript
beforeEach(() => {
  restMock.onGet(networkExchangeRateEndpoint).reply(200, mockedExchangeRate);
});
```